metadata-ingestion/src/datahub/ingestion/source/s3/config.py

```diff
@@ -66,6 +66,11 @@ class DataLakeSourceConfig(
         default="4g", description="Max amount of memory to grant Spark."
     )
 
+    spark_config: Dict[str, Any] = Field(
+        description="Spark configuration properties to set on the SparkSession. Put config property names into quotes. ",
```

Comment:
Could you add a comment with an example `spark_config`, e.g. one that increases the memory